games/ninswitch/game_simple.py

```diff
@@ -1,15 +1,36 @@
 from surrortg import Game
+from surrortg.inputs import Switch
 from games.ninswitch.ns_gamepad_serial import NSGamepadSerial, NSButton, NSDPad
 from games.ninswitch.ns_switch import NSSwitch
 from games.ninswitch.ns_dpad_switch import NSDPadSwitch
 from games.ninswitch.ns_joystick import NSJoystick
+import asyncio
+import logging
+import pigpio
+
+pigpio.exceptions = False
+pi = pigpio.pi()
+nsg_reset = 22
+ON = 0
+OFF = 1
+
+
+class ResetTrinkets(Switch):
```

Comment:
Based on our own games, it should be enough to reset trinkets between the games.

So I would replace the `ResetTrinkets` class with this function:
```python
async def reset_trinket():
    pi.write(nsg_reset, ON)
    logging.info(f"\t{seat} | TRINKET_RESET down")
    pi.write(nsg_reset, OFF)
    logging.info(f"\t...{seat} | TRINKET_RESET up")
    await asyncio.sleep(2)
```

and remove the input registered to it + modify on_prepare.

This simplification would make the `game_simple.py` a bit easier to understand for the creators.

Reply:
In the first weekend running botw, there were probably 10+ times where someone in game had the trinket freeze up on them. If you feel that having input to allow users to manually reset should that happen isn't needed, I will go through with removing it.